bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss

```diff
@@ -76,7 +76,7 @@
 }
 
 .dropdownListContainer {
-  min-width: 10rem;
+  min-width: 18rem;
```

Comment:
instead of fixed widths take a look at this:
width: max-content;

https://caniuse.com/?search=max-content
prefer max width

Reply:
nice, thanks!